plugins/repoquery.py

```diff
@@ -337,7 +413,7 @@ def buildtime(self):
 
     @property
     def conflicts(self):
-        return self._reldep_to_list(self._pkg.conflicts)
+        return self._reldep_to_list(self._pkg.obsoletes)
```

Comment:
This dont look right, look like you are undoing another PR, You will proberly need to rebase you changes to the current HEAD